server/routes/update.test.ts

```diff
@@ -4,26 +4,36 @@ import request from 'supertest'
 import { SanitisedError } from '../sanitisedError'
 import { appWithAllRoutes } from './testutils/appSetup'
 import routeUrls from '../services/routeUrls'
-import { OffenderSearchClient } from '../data/offenderSearch'
-import { davidJones } from '../data/testData/offenderSearch'
+import { NonAssociationsApi } from '../data/nonAssociationsApi'
+import { OffenderSearchClient, OffenderSearchResult } from '../data/offenderSearch'
+import { mockNonAssociation } from '../data/testData/nonAssociationsApi'
+import { davidJones, fredMills } from '../data/testData/offenderSearch'
+import { nameOfPerson } from '../utils/utils'
 
 jest.mock('../data/hmppsAuthClient')
+jest.mock('../data/nonAssociationsApi')
```

Comment:
you should do this to preserve the constants like `roleOptions`

https://github.com/ministryofjustice/hmpps-non-associations/blob/7f6dceb2e4023670d1b2c79afdada1bfdf4bc7b7/server/routes/add.test.ts#L13-L19